crates/composefs/src/erofs/reader.rs

```diff
@@ -175,8 +175,14 @@ impl<Header: InodeHeader> InodeOps for &Inode<Header> {
         }
     }
 
-    fn inline(&self) -> &[u8] {
-        &self.data[self.header.xattr_size()..]
+    fn inline(&self) -> Option<&[u8]> {
+        let data = &self.data[self.header.xattr_size()..];
```

Comment:
Unrelated to this but it made me think that we likely panic if the xattr size is out of bounds.

Of course handling panic freedom here in this whole module probably needs nontrivial work.